scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -890,7 +890,8 @@
                                                             (cache/hit c slave-id)
                                                             (cache/miss c slave-id attrs)))))
                       _ (log/debug "In" pool "pool, passing following offers to handle-resource-offers!" offers)
-                      user->quota (quota/create-user->quota-fn (d/db conn) (name pool))
+                      using-pools? (config/default-pool)
```

Comment:
Nitpick: This name is confusing as `using-pools?` is not a boolean. Can we change it to `(not (nil? (config/default-pool)))`?